pkg/webui/console/views/device-add/wizard/network-settings-form/index.js

```diff
@@ -117,6 +117,7 @@ const NetworkSettingsForm = props => {
         title={sharedMessages.supportsClassC}
         name="supports_class_c"
         component={Checkbox}
+        required={isMulticast}
```

Comment:
```suggestion
```

Reply:
why?

Reply:
Marking checkboxes as required communicates that the option has to be checked consciously in order to continue. This is usually used to ensure that the user is aware of a certain condition or consequence (e.g. agreeing to ToS). In this case, I don't think we need this explicit confirmation and instead, we'll prefill and lock the checkbox field with the mandatory option. Therefore, it's also not necessary to mark the checkbox field as required, just like it also isn't under normal circumstances.